tutorials/vpn-interop-guides/using-cloud-vpn-with-aws/index.md

```diff
@@ -0,0 +1,591 @@
+      - [Creating the Cloud Router](#creating-the-cloud-router)
+      - [Creating the VPN Tunnels ](#creating-the-vpn-tunnels)
+      - [Adding the BGP Link Local Interface ](#adding-the-bgp-link-local-interface)
+      - [Adding the BGP Peering Session](#adding-the-bgp-peering-session)
+- [Testing the Site-to-Site VPN](#testing-the-site-to-site-vpn)
+   - [Verifying Connectivity](#verifying-connectivity)
+   - [Testing the VPN Tunnel](#testing-the-vpn-tunnel)
+   - [Troubleshooting](#troubleshooting)
+
+# Introduction
+
+This guide walks you through the process of configuring the AWS Virtual Private Gateway for integration with the [Google Cloud VPN service](https://cloud.google.com/compute/docs/vpn/).  This information is provided as an example only.  If you are using this guide to configure your AWS implementation, be sure to substitute the correct IP information for your environment.
+
+# Topology
+
+This guide describes the following VPN topologies:  
+
+-  A site-to-site Route based IPsec VPN tunnel configuration. 
+-  A site-to-site IPsec VPN tunnel configuration using the Google Cloud Router and dynamic routing with the BGP protocol.
+
+![image](simple-topology.png)
+
+# Preparation
+
+## Overview
+
+NOTE: The configuration samples in this guide include numerous value substitutions provided only as examples.  When following this guide, replace any references to IP addresses, device IDs, shared secrets, keys, account information, or project names should be replaced with the appropriate values for your environment.
+
+This guide assists you in the creation of IPsec connectivity from AWS to the Google Cloud. The following is a high-level overview of the configuration process:
+
+1. Configure the Amazon Virtual Private Gateway
+1. Configure the Amazon Customer Gateway
+1. Configure the Google Cloud Platform VPN
+1. Set up the VPN Connection
+1. Connect to GCP
+1. Test the tunnel
+
+## Getting Started
+
+The first step is to establish the base networking environment in AWS, which is called the Virtual Private Cloud (VPC).   Amazon provides [documentation](http://aws.amazon.com/documentation/vpc/) for getting started with AWS networking.  The basic concepts to understand are:
+
+-  **Virtual Private Cloud **– a customer-defined private network space in AWS.  
+-  **Virtual Private Gateway** – the VPN concentrator on the Amazon side of the VPN connection.
+-  **Customer Gateway** – an AWS reference to the remote IPsec endpoint.  In this case, the Google Cloud Platform VPN gateway.
+
+## IPsec Parameters 
+
+This table covers IPsec parameters to use when configuring VPN gateways and tunnels as described in this document._ _The IPsec connectivity covered in this guide uses the** pre-shared key** generated by AWS for authentication. AWS only supports IKEv1. Detailed GCP-supported IKEv1 ciphers can be found [here.](https://cloud.google.com/vpn/docs/concepts/advanced#supported_ike_ciphers)
```

Comment:
Prior to 2nd sentence, remove the 2 underscores.
Also, I think you are trying to bold "pre-shared key". Need to a space after "the" and remove the space before "pre"